### DISCUSSION TITLE
not able to activate automation or get automation running in Budibase cloud

### DISCUSSION THREAD

**Discussion by vikrantsharma123** [Budibase/budibase#3127] [Help] [answered]
Hi all,
I am using Budibase Cloud and Budibase Built-in DB. 

I created a simple automation, based on "row created" trigger, I am able to test it as well, but somehow the automation does not run or does not get triggered from the published/running app. Does the automation need to be activated or enabled? I do not see any option to activate/enable the automation. Or is there any other setting somewhere?

Please suggest.

![image](https://user-images.githubusercontent.com/14836167/138580802-7d11f1df-0b59-495b-aa87-9567603f83ad.png)

Thanks and regards,
Vikrant Sharma

**Comment by mjashanks** [comment, marked as answer]
Hi Vikrant.

I have just tested this - it is working for me. 

Automations only run in published apps. So, you need to 
1. Publish your app
2. Go back to the Apps List and click "View Published App"
3. Test your automation

![image](https://user-images.githubusercontent.com/3524181/138682775-dcc30cd5-720a-4e68-a831-e70b7f9ee617.png)

I think we need a quicker way of accessing your published app.

Note that your end users see a different Apps List, which brings them directly to published apps when they click on them. You can see what this looks like by clicking "Close Developer Mode" from your profile icon:

![image](https://user-images.githubusercontent.com/3524181/138683059-073feafe-2073-4d8e-b471-f29cbf8475a7.png)

It's interesting to see that you are trying to build an audit system - this is something that we plan to build directly into Budibase. We won't be doing it in the next few months, but it will hopefully come quite soon.

**Reply by vikrantsharma123** [reply]
Hi Michael,

Yes, it is working fine. Sorry it was a confusion. I was using the "play" button (next to publish) after publishing, to launch the app. Now I used "View Published App", and from there my automation works just fine.

Thanks for your help.

Yes it will be really nice if you can build audit trail (also known as audit log or change log) functionality directly into the system. It will make the app development faster. This is a common requirement in enterprise custom apps, and we spend a lot of time implementing such features.

Regards,
Vikrant Sharma
